pkg/storage/unified/resource/index.go

```diff
@@ -52,7 +53,13 @@ func (i *Index) Init(ctx context.Context) error {
 			if err != nil {
 				return err
 			}
-			err = shard.batch.Index(res.Metadata.Uid, obj)
+
+			var jsonDoc interface{}
+			err = json.Unmarshal(obj.Value, &jsonDoc)
+			if err != nil {
+				return err
+			}
+			err = shard.batch.Index(res.Metadata.Uid, jsonDoc)
```

Comment:
Does indexing a `interface{}` work with Bleve? I thought it needed to be a struct

Reply:
yes.  i'm assuming this is the [preferred way when handling bytes](https://github.com/blevesearch/beer-search/blob/204db6c49802de609443fab595ce08cee5e58b97/main.go#L131).  ...since the author of the lib did it in that example